src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -19,6 +20,24 @@ import { OrganizationFilters } from './OrganizationFilters';
 import { PublisherFilter } from './PublisherFilter';
 import { SeriesFilter } from './SeriesFilter';
 
+const StyledDivider = styled(Divider)(({ theme }) => ({
+  backgroundColor: theme.palette.primary.main,
+}));
+
+const StyledFilterContainer = styled(Box)({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '0.5rem',
+});
+
+const StyledGridItem = styled(Grid)(({ theme }) => ({
+  gap: '1rem',
+  display: 'flex',
+  [theme.breakpoints.down('lg')]: {
+    flexDirection: 'column',
+  },
+}));
```

Comment:
Kanskje prøve denne også da, for å unngå å måtte spesifisere alle plasser? (har bare brukt dette en gang før selv, så sjekk i så fall at det virker i tilfelle du vil gjøre det sånn 😅 )
```suggestion
}));
StyledGridItem.defaultProps = {
  item: true,
};

```